Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/NewMeterWizard.tsx

```diff
@@ -316,6 +316,22 @@ export default function NewMeterWizard(props: {IsEngineer: boolean}) {
         localStorage.setItem('NewMeterWizard.CurrentStep', currentStep.toString())
     }
 
+    function meterDetails() {
+        setError([]);
+        setWarning([]);
+        if (meterID > 0) {
```

Comment:
Is this an issue that can occur? If so that should disable the button and not add an error after it fails.

Reply:
I don't believe this issue can occur, the button should only appear when a meter is created, but I added this error checking if a meter ID happens to be negative.